include/itkTileMerging.h

```diff
@@ -0,0 +1,218 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#ifndef itkTileMerging_h
+#define itkTileMerging_h
+
+#include "itkTileMontage.h"
+#include "itkLinearInterpolateImageFunction.h"
+#include "itkSimpleFastMutexLock.h"
+
+namespace itk
+{
+/** \class TileMerging
+ * \brief Resamples an n-Dimensional mosaic of images into a single big image.
+ *
+ * CropToFill indicates whether the big image will be cropped so it
+ * entirely consists of input tiles (no default background filling).
+ * If CropToFill is false, the big image will have the extent to include
+ * all of the input tiles. The pixels not covered by any input tile
+ * will have the value specified by the Background member variable.
```

Comment:
`Background` -> `BackgroundValue` to be more explicit and follow ITK naming conventions